test/appium/tests/atomic/dapps_and_browsing/test_dapps.py

```diff
@@ -38,3 +38,25 @@ def test_request_public_key_status_test_daap(self):
         status_test_dapp.allow_button.click()
         if not status_test_dapp.element_by_text(user['public_key']).is_element_displayed():
             pytest.fail('Public key is not returned')
+
+    @marks.testrail_id(5654)
+    @marks.low
+    def test_can_proceed_dapp_usage_after_transacting_it(self):
+        user = basic_user
+        sign_in_view = SignInView(self.driver)
+        home_view = sign_in_view.recover_access(passphrase=user['passphrase'])
+        wallet = home_view.wallet_button.click()
+        wallet.set_up_wallet()
```

Comment:
Let move "Wallet Set Up" after Assets button is taped ( `4) Tap Request STT. Finish Wallet Set Up.` )

Reply:
@hetvart please move the setup according to the comment